packages/venia-ui/lib/components/Header/searchTrigger.css

```diff
@@ -1,13 +1,36 @@
+:root {
+    --search-triger-width: 3rem;
+}
 .root {
     composes: root from '../clickable.css';
     transition-property: color;
     transition-duration: 224ms;
     transition-timing-function: cubic-bezier(0, 0, 0.2, 1);
     height: 3rem;
-    width: 3rem;
+    width: var(--search-triger-width);
 }
 
 .open {
     composes: root;
     color: rgb(var(--venia-global-color-teal));
```

Comment:
While we're in this file, let's swap this over to the brand color property.

Reply:
Done